interface/app/$libraryId/Explorer/QuickPreview/index.tsx

```
@@ -396,6 +439,16 @@ export const QuickPreview = () => {
 
 								<FileThumb
 									data={item}
+									onLoad={(type) =>
+										type === 'ORIGINAL' && setThumbErrorToast(undefined)
+									}
+									onError={(type, error) =>
+										type === 'ORIGINAL' &&
+										setThumbErrorToast({
+											title: 'Error loading original file',
+											body: error.message
+										})
+									}
```

Can we not trigger a toast from here with `toast.error `instead?

---

We would need a state anyway to allow closing the toast whenever the quick preview is dismissed and the user didn't close the toast manually, so I found it to be a better approach to isolate all the show/hide logic for the toast into a `useEffect`, and just allow the component to `setThumbErrorToast` whenever it needed to show/hide an error toast.